index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  dirname: __dirname,
```

Comment:
I did it this way because it's the most extensible not knowing our future
needs.

On 7 Sep 2016 2:23 AM, "Nick Schonning" <notifications@github.com> wrote:

> In index.js
> <https://github.com/sass/sass-spec/pull/906#discussion_r77668722>:
>
> > @@ -0,0 +1,3 @@
> > +module.exports = {
> > +  dirname: __dirname,
>
> Fine with the current, but maybe specPath = path.join(__dirname, 'spec')
> might be more usefull in the long run
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sass/sass-spec/pull/906/files/229326a3c10959462cd3041a66d36fce09ac07ee#r77668722>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAjZWFbCj9i8CawRns6N_OWa67uErcZdks5qnZNDgaJpZM4J1u0b>
> .
>